src/snowflake/snowpark/dataframe.py

```diff
@@ -1218,15 +1215,15 @@ def agg(
             ----------------------
             <BLANKLINE>
 
-            >>> df.agg([stddev(col("a")), stddev_pop(col("a"))]).show()
+            >>> df.agg(stddev(col("a")), stddev_pop(col("a"))).show()
             -------------------------------------------
             |"STDDEV(A)"         |"STDDEV_POP(A)"     |
             -------------------------------------------
             |1.1547003940416753  |0.9428091005076267  |
             -------------------------------------------
             <BLANKLINE>
 
-            >>> df.agg([("a", "min"), ("b", "max")]).show()
+            >>> df.agg(("a", "min"), ("b", "max")).show()
```

Comment:
As per discussion offline, it's better to support the bracket too.
```
df.agg(["a", "min"], ["b", "max"]).show()
```

Reply:
done